packages/frontend/src/utils/wallet.js

```diff
@@ -860,7 +860,7 @@ class Wallet {
         accountIds = [...accountsSet];
 
         if (!accountIds.length) {
-            throw new WalletError('Cannot find matching public key', 'recoverAccountSeedPhrase.errorInvalidSeedPhrase', { publicKey });
```

Comment:
Originally I looked at why the error parsing/alert box wasn't displaying this but after revisiting the original issue I noticed that the first parameter to `WalletError` is what appears in the code-styled box on the alert surfaced to the user.